src/client/components/Speksit/Speksit.css

```diff
@@ -1,93 +1,77 @@
 .container {
-  background-color: #fff;
+  background-color: #a9a9a9;
   width: 100%;
 }
 
-.hero {
-  display: flex;
-	padding-top: 100px;
-	padding-bottom: 30px;
-  background-color: #fff;
+.year {
   width: 100%;
+  display: flex;
+  composes: subHeading from '../App.css';
+  justify-content: flex-end;
+  padding: 2em 4em;
 }
 
-.heroSpeksit {
-
-}
-
-.heroFirstColumn {
+.speksiContainer {
+  margin-top: 1em;
   display: flex;
   flex-direction: column;
-  width: 100%;
+  flex-wrap: wrap;
   justify-content: center;
+  align-items: center;
+  border-bottom: 1px black solid;
 }
 
-.heroSecondColumn {
+.speksiTitle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: 100%;
+  padding: 2em;
 }
 
-.heroTextColumn {
-  padding: 2em;
+.carouselContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: auto;
 }
 
-.carousel {
-	width: 100%;
-	height: auto;
-	margin: 0px;
-	padding: 0px;
-	padding-bottom: 30px;
+.speksiDescription {
+  width: 80%;
+  padding: 100px 2em;
 }
 
-/* STYLES FOR ALA AMMU OHI */
+.black {
+  /* background-color: black; */
+}
 
-.content_alaammuohi {
-	background-color: #FFECAF;
-	margin: 0px;
-	padding: 0px;
-	padding-top: 20px;
-	text-align: left;
+.whiteText {
+  /* color: white; */
 }
+
 .font_alaammuohi_header {
```

Comment:
Should the style names be changed to match the show's year instead of name?

Reply:
Could be, but I think that isn't about this pr and can be done later when refactoring.